packages/ethereum-contracts/contracts/interfaces/superfluid/ISuperfluidGovernance.sol

```diff
@@ -41,7 +41,8 @@ interface ISuperfluidGovernance {
         ISuperfluid host,
         address hostNewLogic,
         address[] calldata agreementClassNewLogics,
-        address superTokenFactoryNewLogic
+        address superTokenFactoryNewLogic,
+        address beaconNewLogic
```

Comment:
@d10r we need to upgrade the governance contract, right?

Reply:
yes